src/mwindow.c

```diff
@@ -201,10 +201,6 @@ static bool git_mwindow_scan_recently_used(
 	assert(mwf);
 	assert(out_window);
 
-	lru_window = *out_window;
-	if (out_last)
-		lru_last = *out_last;
-
```

Comment:
is there a way in which this bit is preserved? the reason why it's here is that the caller can provide a reference window that all the other windows in `mwf` will be compared against (and maybe add this information as a comment for future spelunkers).

Reply:
Ah, I didn't spot that the initial value would be compared against! I've added these lines back and used a flag instead